### DISCUSSION TITLE
An action starts 3 months ago still running

### DISCUSSION THREAD

**Discussion by kkocdko** [community/community#45572] [Actions] [answered]
### Select Topic Area

Bug

### Body

https://github.com/r-darwish/topgrade/actions/runs/3234515441/jobs/5297707193

![image](https://user-images.githubusercontent.com/31189892/215181474-c4799343-8503-4d11-bb81-00c9ed4733bf.png)

So funny.

**Comment by jge162** [comment, marked as answer]
It is possible that the action is still running on GitHub if it was set up to run on a schedule or as part of a workflow. If this is the case, it will continue to run until it is manually stopped or it reaches the end of its designated schedule

**Comment by IceflowRE** [comment]
https://github.com/IceflowRE/go-dslp/actions/runs/832439019/jobs/801158538

21 month starting now.
Should i stop it?
